modules/oceanbase/src/main/java/org/testcontainers/oceanbase/OceanBaseCEContainer.java

```diff
@@ -43,6 +68,31 @@ public OceanBaseCEContainer(DockerImageName dockerImageName) {
         dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
 
         addExposedPorts(SQL_PORT, RPC_PORT);
+        setWaitStrategy(Wait.forLogMessage(".*boot success!.*", 1));
+    }
+
+    @Override
+    protected void configure() {
+        addEnv("MODE", mode.name().toLowerCase());
+
+        if (!DEFAULT_USER_TENANT_NAME.equals(tenantName)) {
+            if (mode == Mode.SLIM) {
+                logger().warn("The tenant name is not configurable on slim mode, so this option will be ignored.");
+                // reset the tenant name to ensure the constructed username is correct
+                tenantName = DEFAULT_USER_TENANT_NAME;
+            } else {
+                addEnv("OB_TENANT_NAME", tenantName);
+            }
+        }
+
+        if (!DEFAULT_PASSWORD.equals(password)) {
+            addEnv("OB_TENANT_PASSWORD", password);
+        }
```

Comment:
I think having this is ok because password by default is DEFAULT_PASSWORD

```suggestion
        addEnv("OB_TENANT_PASSWORD", password);
```